src/Admin/PostAdmin.php

```diff
@@ -36,6 +36,8 @@
 class PostAdmin extends AbstractAdmin
 {
     /**
+     * @deprecated since sonata-project/news-bundle 3.x, to be removed in 4.0.
+     *
      * @var UserManagerInterface
```

Comment:
UserManagerInterface|null ?

I mean what if it’s ignored on initialize?

Reply:
Fixed